### PR TITLE
LPS-139430 As an Administrator, I want to assign a User to Accounts

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke @pei-jung 

https://issues.liferay.com/browse/LPS-139430

### CODE REVIEW

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryUserRelLocalServiceTest.java [hunk 1]
```diff
@@ -542,6 +542,22 @@ public void testGetAccountEntryUserRelsByAccountEntryId() throws Exception {
...
+		_accountEntryUserRelLocalService.addAccountEntryUserRel(
+			_accountEntry.getAccountEntryId(), user1.getUserId());
+
+		Assert.assertTrue(
+			_accountEntryUserRelLocalService.isAccountEntryUser(
+				user1.getUserId()));
+		Assert.assertFalse(
+			_accountEntryUserRelLocalService.isAccountEntryUser(
+				user2.getUserId()));
+	}
```
Comment (drewbrokke) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryUserRelLocalServiceTest.java:559]: This is fine, but to make it a little more concise, you can just assert with one user before and after assignment.

```
add user

assert false

assign user to account

assert true
```

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/UpdateAccountMembershipsMVCActionCommand.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	@Override
+	protected void doProcessAction(
+			ActionRequest actionRequest, ActionResponse actionResponse)
+		throws Exception {
+
+		long[] addAccountEntryIds = ParamUtil.getLongValues(
+			actionRequest, "addAccountEntryIds");
+		long[] deleteAccountEntryIds = ParamUtil.getLongValues(
+			actionRequest, "deleteAccountEntryIds");
+
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/UpdateAccountMembershipsMVCActionCommand.java:53]: SF remove this line break. You can retrieve the parameters in any order.

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/dao/search/AccountUserAccountEntryRowChecker.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		if (Validator.isNotNull(getAllRowIds())) {
+			sb.append(
+				getOnClick(
+					checkBoxRowIds, checkBoxAllRowIds, checkBoxPostOnClick));
+		}
+
+		sb.append(">");
+
+		return sb.toString();
+	}
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/dao/search/AccountUserAccountEntryRowChecker.java:107]: Is this custom markup still needed or does setting data on the rows take care of it?

**Review by drewbrokke — APPROVED** [review]
@alee8888 overall this looks good. There are a few changes needed at this point. Also, I'll go over the JS portion in more detail tomorrow, but my first impression is that we should use a few more of the udpated JS tools (I see new usage of AUI, which we generally try to avoid).

Thanks!

**Comment by drewbrokke** [comment]
Sorry @alee8888 I accidentally approved the changes instead of "request changes", but you can see the comments :)

**Comment by epark18** [comment]
ci:test:relevant

**Comment by epark18** [comment]
Hello @alee8888 There is just one CI failure that has to do with a version increase. [Error Logs](https://testray.liferay.com/reports/production/logs/2022-01/test-1-34/test-portal-acceptance-pullrequest(master)/670/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz)
`     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.account.service                        MINOR      5.3.0      5.3.0      5.4.0      VERSION INCREASE REQUIRED
     [exec] 	<   class      com.liferay.account.service.AccountEntryUserRelLocalServiceUtil
     [exec] 		+   method     isAccountEntryUser(long)
     [exec] 			+   access     static
     [exec] 			+   return     boolean
     [exec] 	<   class      com.liferay.account.service.AccountEntryUserRelLocalServiceWrapper
     [exec] 		+   method     isAccountEntryUser(long)
     [exec] 			+   return     boolean
     [exec] 	<   interface  com.liferay.account.service.AccountEntryUserRelLocalService
     [exec] 		+   method     isAccountEntryUser(long)
     [exec] 			+   access     abstract
     [exec] 			+   annotated  com.liferay.portal.kernel.transaction.Transactional
     [exec] 				+   property   propagation='SUPPORTS'
     [exec] 				+   property   readOnly=true
     [exec] 			+   return     boolean
     [exec] [Baseline Warning] Bundle Version Change Recommended: 6.4.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.account.api-6.3.1.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.account.api/6.3.0/92ddd1bb71b0e667163874437fe92cb2e78c4145/com.liferay.account.api-6.3.0.jar`
Thank you!

**Comment by drewbrokke** [comment]
See https://github.com/liferay-usm/liferay-portal/pull/424

**Pull request closed without merging**